INSTALL

```diff
@@ -166,6 +166,11 @@ environment variables are set properly:
     # makes HOME and PYTHONPATH available for python
     runjob --exp-env HOME --exp-env PYTHONPATH ... : python script.py
 
+Compiling NEST with the GCC (`-DCMAKE_TOOLCHAIN_FILE=Platform/BlueGeneQ_GCC`)
+might require you to use a GSL library compiled by the GCC, otherwise undefined
+symbols break your build. Changing the compiler should be done in the file
+`<src>/cmake/Platform/BlueGeneQ_GCC.cmake`.
```

Comment:
I think I just got totally confused because I thought that the second sentence explained how to do what the first sentence suggests. But first and second sentence have nothing to do with each other! Could you add after the first sentence another one  such as "Use `-D<whatever is needed to select the correct GSL>` to link NEST against the correct GSL version. Then start a new paragraph "If you want to use a different version of GCC, please edit ...". I think that will make things more transparent.